install.sh

```diff
@@ -7,10 +7,13 @@
 
 set -euo pipefail
 
+NO_HOOK_NAME="no-hook"
```

Comment:
have you also looked at how the uninstallation would behave? Would any code changes need to be made to the uninstallation process?